pmlc/rt/boundscheck.cc

```diff
@@ -17,13 +17,13 @@ extern "C" void _mlir_ciface_plaidml_rt_bounds_check(intptr_t index,
                              "mlir::memref::StoreOp");
 }
 
-namespace {
-struct Registration {
-  Registration() {
-    pmlc::rt::registerSymbol(
-        "_mlir_ciface_plaidml_rt_bounds_check",
-        reinterpret_cast<void *>(_mlir_ciface_plaidml_rt_bounds_check));
-  }
-};
-static Registration reg;
-} // namespace
+namespace pmlc::rt {
+
+void registerBoundsCheck() {
+  using pmlc::rt::registerSymbol;
```

Comment:
Why do a `using` here if we just use the name once?

Reply:
I was just following the same format used elsewhere (eg registerPrng) but happy to remove it